### PR TITLE
[image_geometry] Enable python package installation

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

**Review by dirk-thomas — APPROVED** [review]

File: image_geometry/CMakeLists.txt [hunk 1]
```diff
@@ -22,9 +25,10 @@ add_library(${PROJECT_NAME}
   src/pinhole_camera_model.cpp src/stereo_camera_model.cpp
 )
 target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES})
-ament_target_dependencies(${PROJECT_NAME}
-  "OpenCV"
-  "sensor_msgs"
+ament_target_dependencies(
+  ${PROJECT_NAME}
+  OpenCV
+  sensor_msgs
```
Comment (wjwwood) [inline image_geometry/CMakeLists.txt:31]: Does this actually change anything?

**Review by wjwwood — APPROVED** [review]

File: image_geometry/CMakeLists.txt (hunk 1 above)
Comment (dirk-thomas) [reply image_geometry/CMakeLists.txt:31]: No.

Comment (mjcarroll) [reply image_geometry/CMakeLists.txt:31]: I thought the preferred style was without quotes, is that true?

Comment (wjwwood) [reply image_geometry/CMakeLists.txt:31]: Historically, I don't think so, but maybe the trend has changed 😅

Comment (wjwwood) [reply image_geometry/CMakeLists.txt:31]: Really doesn't matter though.

**Review by chapulina — APPROVED** [review]

**Pull request merged**